app/models/manageiq/providers/vmware/cloud_manager/orchestration_template.rb

```diff
@@ -29,7 +31,64 @@ def vapp_parameters
     ]
   end
 
+  def vm_param_groups
+    # Parse template's OVF file
+    ovf_doc = MiqXml.load(content)
+
+    groups = []
+    # Parse the XML template document for specific vCloud attributes.
+    ovf_doc.root.each_element("//vcloud:GuestCustomizationSection") do |el|
+      vm_id = el.elements["vcloud:VirtualMachineId"].text
+      vm_name = el.elements["vcloud:ComputerName"].text
+
+      groups << OrchestrationTemplate::OrchestrationParameterGroup.new(
+        :label      => vm_name,
+        :parameters => [
+          # Name of the provisioned instance.
+          OrchestrationTemplate::OrchestrationParameter.new(
+            :name          => "instance_name-#{vm_id}",
+            :label         => "Instance name",
+            :data_type     => "string",
+            :default_value => vm_name
+          ),
+
+          # List of available VDC networks.
+          OrchestrationTemplate::OrchestrationParameter.new(
+            :name          => "vdc_network-#{vm_id}",
+            :label         => "Network",
+            :data_type     => "string",
+            :default_value => "(default)",
```

Comment:
I am still not sure whether I should be using i18n here or not. Other templates will output whatever there is in the template itself, but this is a special case and I believe we could use it...